virtualizarr/writers/icechunk.py

```diff
@@ -0,0 +1,194 @@
+from typing import TYPE_CHECKING, cast
+
+import numpy as np
+from xarray import Dataset
+from xarray.backends.zarr import encode_zarr_attr_value
+from xarray.core.variable import Variable
+from zarr import Group  # type: ignore[import-untyped]
```

Comment:
Currently virtualizarr doesn't directly use zarr, it only needs it if either kerchunk or icechunk is installed. The `min-deps` CI is failing because that doesn't use either, and therefore doesn't install `zarr`. So this import should be protected.

Reply:
Yup this was a mistake. An auto import from the ide. Thanks for catching